config/locales/models.de.yml

```diff
@@ -887,6 +887,8 @@ de:
         participant_number_internal: Teilnehmernummer intern
         payment_reminder_configs: Mahnungen
         vat_number: MwSt. Nummer
+        donation_increase_percentage: Erhöhung in %
+        donation_calculation_year_amount: Anzahl Jahre
```

Comment:
Wie wäre es mit etwas ähnlichem wie:
> Zahlungen aus den letzten [____________] Jahren berücksichtigen.

Wobei [___________] das Input-Feld ist? Oder allenfalls ein erklärender Satz in `help-inline` oder `help-block` wie bei Schulklasse bzw. Zusätzliche Angaben hier:
![Screenshot from 2021-08-13 15-02-11](https://user-images.githubusercontent.com/7566995/129360975-ff0ce7a8-1204-4e67-aa04-e63ede5caf56.png)